### DISCUSSION TITLE
Beginner

### DISCUSSION THREAD

**Discussion by Biggerguy8** [community/community#127415] [New to GitHub]
I have questions to anyone who can help me just get a start, and where to begin to learn. I understand the whole process of how it works, I just need some tips I guess you could sat

**Comment by Sbiswas001** [comment]
what languages do you know
try creating a portfolio website using html, css, js

**Comment by francoriba** [comment]
That would depend on what topics/technologies you are interested in. I would focus on lerning basic programming concepts first, like data structures, some basic dessing patterns, OOP. If you like web development maybe a good project to start is [the odin project](https://www.theodinproject.com/).

**Comment by mkeaton08** [comment]
I would suggest starting off with learning Python. It has clear and readable syntax to help you learn essential programming concepts such as variables, data types, loops, conditionals, functions, and object-oriented programming (OOP) principles. There are websites out there that will allow you to practice coding for free to help learn the basics.

**Comment by rockett-m** [comment]
start with `leetcode.com` and progress from easy problems to mediums

**Comment by Ammar-Alnagar** [comment]
I would say create a good readme file and have a couple of repos under your name .
Then start learning about actions and workflows .
